python/tvm/relay/quantize/_annotate.py

```diff
@@ -166,7 +166,7 @@ def multiply_rewrite(ref_call, new_args, ctx):
 
     if lhs_kind is None and rhs_kind is None:
         return None
-    if lhs_kind == QAnnotateKind.ACTIVATION and rhs_kind is None:
+    if lhs_kind in [QAnnotateKind.ACTIVATION, QAnnotateKind.INPUT] and rhs_kind is None:
```

Comment:
add a check to `attach_simulated_quantize` below to prevent quantizing lhs to INPUT if lhs_kind is INPUT

Reply:
I'm not quite familiar with this, but what shall `lhs_expr` be if `lhs_kind` is INPUT?

Reply:
it is the same from input, `new_args[0]`

Reply:
okay, i will revise accordingly.